database/src/lib.rs

```diff
@@ -309,9 +311,16 @@ pub trait ScyllaStorageManager {
     async fn prepare_query(
         scylla_db_session: &std::sync::Arc<scylla::Session>,
         query_text: &str,
+        consistency: Option<scylla::frame::types::Consistency>,
```

Comment:
What does consistency mean for read queries?

Reply:
My picture now: we should always use `quorum` (actually, even `all`) for read queries because there's a chance that we don't have data on some of our instances. It will provide us with the errors.

We may always use `any` for write queries to speed up the solution. Though other consistency levels are fine, it will just slow down the writing process.

Correct me if I'm wrong.
But, even if we have another picture, my concern here is that we should keep this all in mind during developing new features and reviewing the code. I suggest making wrappers, if it's possible (`prepare_write_query`, `prepare_read_query`).